tgstation.dme

```diff
@@ -1718,11 +1718,7 @@
 #include "code\modules\antagonists\abductor\equipment\glands\electric.dm"
 #include "code\modules\antagonists\abductor\equipment\glands\heal.dm"
 #include "code\modules\antagonists\abductor\equipment\glands\mindshock.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\plasma.dm"
 #include "code\modules\antagonists\abductor\equipment\glands\quantum.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\slime.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\spider.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\transform.dm"
```

Comment:
dont remove these files, you may cause issues later down the line if these organs are used elsewhere, simply commenting out the organs from being in rotation are fine as they are, removing the organs completely is just unnesscary